src/engine/DnCManager.h

```diff
@@ -53,7 +53,7 @@ class DnCManager
     /*
       Perform the Divide-and-conquer solving
     */
-    void solve( unsigned timeoutInSeconds );
+    void solve( unsigned timeoutInSeconds, bool restoreTreeStates=true );
```

Comment:
Spacing: restoreTreeStates=true --> restoreTreeStates = true

Also, not crucial, but it's strange that the default configuration is false, but the default parameter to the function is true.

Reply:
Fixed spacing; set default to false.